### PR TITLE
WM3 Halla Sulaiman/ Git and Github

### PR DESCRIPTION
**Pull request by Halla-S**
# Your Details

- Your Name: Halla Sulaiman
- Your City: Wolverhampton
- Your Slack Name: Halla Sulaiman

# Homework Details

- Module: Git and Git Hub
- Week: week 2 Full Stack

-----
[View rendered homework.md](https://github.com/Halla-S/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

**Comment by justsealan** [comment]
Good job 👏 👏

File: homework.md [hunk 1]
```diff
@@ -44,43 +43,43 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-When you send a file over the internet
+A method of submitting contributions to the project .
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see The whole changes of the repository.
```
Comment (PeterAhern) [inline homework.md:76]: It let's you review each and every commit made to the repository

**Review by PeterAhern — COMMENTED** [review]
Well done! You just need to add the Week number at the end of your Pull Request Title.
It is either Week 1 or Week 2, I thought it was week 1, but some others think it is week 2.

**Comment by Krrish-CYF** [comment]
Hi Halla, I think you've explained everything's so clearly well done!

File: homework.md [hunk 2]
```diff
@@ -44,43 +43,43 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-A programming language used on the internet
+A programming language for the web
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Git is a version control . It keeps track of projects , and allows many collaborors to work on a project at the same time .
```
Comment (gynaa78) [inline homework.md:52]: Yes, it allows many **collaborators** to work on a project.

**Review by gynaa78 — COMMENTED** [review]
Good job! Kudos.

File: homework.md (hunk 2 above)
Comment (Aclayonwright) [reply homework.md:52]: Its great for collaborating

**Comment by mimi2111** [comment]
great job Halla

**Pull request closed without merging**